tests/integration_test/validators/sag_result_validator.py

```diff
@@ -51,6 +51,42 @@ def check_sag_results(server_data, run_data):
     return True
 
 
+def ha_check_sag_results(server_data, run_data):
```

Comment:
if this logic is the same as "SagResultValidator" maybe we just use that for now?

Reply:
The results are different because we couldn't stop the server at exact same location.  It runs one more round.